dynamiqs/utils/utils.py

```diff
@@ -735,3 +735,24 @@ def mpow(x: Tensor, n: int) -> Tensor:
                 [0.+0.j, 1.+0.j]])
     """
     return torch.linalg.matrix_power(x, n)
+
+def entropy_vn(x: Tensor) -> Tensor:
```

Comment:
When we define a new function, we need to add it in three places:
- In the variable `__all__` at the top of the file.
- In the file `dynamiqs/mkdocs.yml` (cmd + p to look for file on vscode, then search for the `mpow` function to take an example)
- Due to a recent change we also need to add it to `dynamiqs/python_api/index.md` (it's the file that renders the table here https://www.dynamiqs.org/python_api/index.html). However you started your PR _before_ this change was merged to main. To catch the new changes and update your branch to the latest version you need to (1) checkout the branch `main` (make sure you don't have work in progress or you won't be able to switch branch), (2) pull the branch `main` to get the latest changes, (3) checkout your branch `Von-Neumann` and (4) `git merge main` to merge the changes from main. At this step if you have conflicts or issues you can call me 😉 And then you can add your new function to `dynamiqs/python_api/index.md` so it appears correctly in the API table.

I will document all this very soon in the `CONTRIBUTING.md`.